hbase-server/src/test/java/org/apache/hadoop/hbase/master/normalizer/TestSimpleRegionNormalizer.java

```diff
@@ -448,6 +448,38 @@ public void testMergeEmptyRegions1() {
         .build()));
   }
 
+  @Test
+  public void testMergeEmptyRegions2() {
+    conf.setBoolean(SPLIT_ENABLED_KEY, false);
+    conf.setInt(MERGE_MIN_REGION_SIZE_MB_KEY, 0);
+    final TableName tableName = name.getTableName();
+    final List<RegionInfo> regionInfos = createRegionInfos(tableName, 8);
+    final Map<byte[], Integer> regionSizes =
+      createRegionSizesMap(regionInfos, 0, 10, 1, 0, 9, 0, 10, 0);
+    setupMocksForNormalizer(regionSizes, regionInfos);
+
+    assertFalse(normalizer.isSplitEnabled());
+    assertEquals(0, normalizer.getMergeMinRegionSizeMb());
+    List<NormalizationPlan> plans = normalizer.computePlansForTable(tableName);
+    assertThat(plans, contains(
+      new MergeNormalizationPlan.Builder()
```

Comment:
Without source code changes, the first pair would be {10, 1} right?

Reply:
Without source code changes, the first pair would be {1, 0}, because 10 is bigger than avgSize.